MonoGame.Framework/Graphics/GraphicsAdapter.cs

```diff
@@ -107,6 +111,12 @@ public DisplayMode CurrentDisplayMode
                     graphics.ReleaseHdc(dc);
                     return new DisplayMode(width, height, SurfaceFormat.Color);
                 }
+#elif WINDOWS_UAP
+                var visibleBounds = ApplicationView.GetForCurrentView().VisibleBounds;
+                var rawPixelsPerViewPixel = DisplayInformation.GetForCurrentView().RawPixelsPerViewPixel;
+                return new DisplayMode((int)(visibleBounds.Width * rawPixelsPerViewPixel + 0.5f),
+                                       (int)(visibleBounds.Height * rawPixelsPerViewPixel + 0.5f),
+                                       SurfaceFormat.Color);
 #else

```

Comment:
Well it is a trick, metro style apps take the full screen when they start up. Microsoft decided to drop exclusive access to screen - no fullscreen, no vertical sync disable, no information about native screen size, etc. It is really boring to develop games with c# now. If i had the knowledge from today when i started with XNA i would have had droped it and change to an other framework and language, maybe OS too. Metro Style + NetCore is a real wrong way, do things complicated without any need and a lot less possibilities and comfort. I am really upset of MS C# misleading the users without any native support for games programming anymore. But now i have a big engine and i do not want to drop that hard work - currently. If i would have the time i would give Rust a better try.

https://social.msdn.microsoft.com/Forums/en-US/e3c4d339-a6aa-42e0-a453-3d6eddeca5d9/get-physical-screen-size?forum=winappswithcsharp